apps/nextjs/src/components/DownloadButton.tsx

```diff
@@ -46,8 +46,11 @@ export const DownloadButton = ({
 
   const { icon, ext, analyticsResourceType } = getExportsConfig(exportsType);
 
-  const { mutateAsync, isLoading } =
-    trpc.exports.sendUserExportLink.useMutation();
+  const {
+    data: sendUserExportLinkSent,
```

Comment:
Could you try using `isSuccess` instead? That should avoid the need to convert to a boolean